tests/functional/c/consider/consider_using_str_partition.txt

```diff
@@ -0,0 +1,20 @@
+consider-using-str-partition:5:12::Consider using '1,2,3'.partition(',')[0] instead
+consider-using-str-partition:6:11::"Consider using '1,2,3'[::-1].partition(',')[0] instead"
+consider-using-str-partition:9:12::Consider using SEQ.partition(',')[0] instead
+consider-using-str-partition:10:11::Consider using SEQ.rpartition(',')[-1] instead
```

Comment:
```suggestion
consider-using-str-partition:10:11::Consider using SEQ.rpartition(',')[2] instead
```
You probably need to adjust the second `elif` case and change `subscript_value` from `-1` to `2`.

Reply:
I feel that this might be opinionated too

Code in question:
```
get_last = SEQ.split(',')[-1]  # [consider-using-str-partition]
```

I would consider the suggestion
```
consider-using-str-partition:10:11::Consider using SEQ.rpartition(',')[-1] instead
```

to be better in my opinion here